web/src/components/tooltips/mapcountrytooltip.js

```diff
@@ -12,9 +13,16 @@ const mapStateToProps = state => ({
   electricityMixMode: state.application.electricityMixMode,
 });
 
+
 const TooltipContent = React.memo(({
   isDataDelayed, hasParser, co2intensity, fossilFuelPercentage, renewablePercentage,
 }) => {
+  const CountryTableHeaderInner = styled.div`
+    display: flex;
+    flex-basis: 33.3%;
+    justify-content: space-between;
+  `;
+
```

Comment:
This styled component must be created outside of `TooltipContent`:

<img width="566" alt="Screenshot 2021-09-09 at 09 03 06" src="https://user-images.githubusercontent.com/3296643/132638585-6bd97ccc-2b36-434a-a177-0366141f3d3c.png">